### ISSUE TITLE
Octavious Ward custom items

### ISSUE THREAD

**Issue by Tasald** [VOREStation/VOREStation#1129]
#### Brief description of the issue
Muh snowflake items not in/ extremely broken

#### What you expected to happen
To log in as my person and have the box of goodies.

#### What actually happened
Spawned in, had a box of stuff but it was all basically broken and unusable, along with some things that were missing.

#### Steps to reproduce
Log on as Octavious ward
Look in box and see HONK
~~Get band for impersonating one of my characters.~~

#### Code Revision
Server revision: release - 2017-03-07
2b0d55d72f3642c602025608c1bf691a25381a36 
 
#### Anything else you may wish to add:
basically all but the hat are not in/broken to hell, and the hat has one 'error'? when you turn side ways the ears disappear for me.

Would it be possible to make the monocle a reskinned medihud?

Here is the links to all the item apps from the forum.

http://forum.vore-station.net/viewtopic.php?f=26&t=600
http://forum.vore-station.net/viewtopic.php?f=26&t=654
http://forum.vore-station.net/viewtopic.php?f=26&t=604
http://forum.vore-station.net/viewtopic.php?f=26&t=603
http://forum.vore-station.net/viewtopic.php?f=26&t=602
http://forum.vore-station.net/viewtopic.php?f=26&t=601

**Comment by JoanRisu** [comment]
For some reason the mask code isn't cooperating with me.

**Comment by Tasald** [comment]
What about the other items getting added?

**Comment by JoanRisu** [comment]
I don't have the sprites for the other items.
